browser/skus/skus_sdk_mojom_impl.h

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_BROWSER_SKUS_SKUS_SDK_MOJOM_IMPL_H_
+#define BRAVE_BROWSER_SKUS_SKUS_SDK_MOJOM_IMPL_H_
+
+#include <string>
+
+#include "brave/components/skus/common/skus_sdk.mojom.h"
+
+class SkusSdkService;
+
+namespace content {
+class BrowserContext;
+}  // namespace content
+
+namespace brave_rewards {
+
+class SkusSdkMojomImpl final : public skus::mojom::SkusSdk {
+ public:
+  SkusSdkMojomImpl(const SkusSdkMojomImpl&) = delete;
+  SkusSdkMojomImpl& operator=(const SkusSdkMojomImpl&) = delete;
+
+  explicit SkusSdkMojomImpl(content::BrowserContext* context);
```

Comment:
iOS can't use this class because its dependent on `content::BrowserContext *`

Reply:
I don't think iOS should use this class though - so we should be good 😄 We'll want to use the `SkuSdkService` basically (this is specifically providing a DOM implementation for window.chrome.braveSkus.refresh_order` (or similar)

Reply:
We cannot use `SkuSdkService` because it does not conform to `skus::mojom::SkusSdk`

Reply:
https://github.com/brave/brave-core/pull/10847/files#r762257172

Reply:
This is impl class was removed; addressed with https://github.com/brave/brave-core/commit/0c492c6a95c3b11eb0f4ab4c316510b78a030d19 /  https://github.com/brave/brave-core/pull/10847/commits/791f0b03062acdb751c210042bb3079b338ea051 / https://github.com/brave/brave-core/pull/10847/commits/2dacad8e9efaca7acf15e66420ba435fdc886644